common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -138,40 +142,42 @@ def test_hidden_content_before_due(self):
         self.assertIn("seq_module.html", html)
         self.assertIn("'banner_text': None", html)
 
-    @freeze_time(DAY_AFTER_TOMORROW)
-    @ddt.data(
-        (None, 'subsection'),
-        ('Homework', 'homework'),
-    )
-    @ddt.unpack
-    def test_hidden_content_past_due(self, format_type, expected_text):
+    @freeze_time(PAST_FUTURE_DATE)
+    def test_hidden_content_past_due(self):
         progress_url = 'http://test_progress_link'
-        self._set_sequence_format(self.sequence_4_1, format_type)
         html = self._get_rendered_student_view(
             self.sequence_4_1,
             extra_context=dict(progress_url=progress_url),
         )
         self.assertIn("hidden_content.html", html)
         self.assertIn(progress_url, html)
-        self.assertIn("'subsection_format': '{}'".format(expected_text), html)
 
-    @freeze_time(DAY_AFTER_TOMORROW)
+    @freeze_time(PAST_FUTURE_DATE)
     def test_masquerade_hidden_content_past_due(self):
-        self._set_sequence_format(self.sequence_4_1, "Homework")
         html = self._get_rendered_student_view(
             self.sequence_4_1,
             extra_context=dict(specific_masquerade=True),
         )
         self.assertIn("seq_module.html", html)
         self.assertIn(
             "'banner_text': 'Because the due date has passed, "
-            "this homework is hidden from the learner.'",
+            "this assignment is hidden from the learner.'",
             html
         )
 
-    def _set_sequence_format(self, sequence, format_type):
-        """
-        Sets the format field on the given sequence to the
-        given value.
-        """
-        sequence._xmodule.format = format_type  # pylint: disable=protected-access
+
+class SelfPacedSequenceBlockTestCase(SequenceBlockTestBase):
```

Comment:
Can you add a positive test to make sure the content isn't hidden before the course-end-date for self-paced courses: `test_hidden_content_before_end`?